frontend/app/src/premium/register-components.ts

```diff
@@ -126,6 +127,7 @@ export function registerComponents(app: App): void {
   app.component('UniswapPoolAssetBalance', UniswapPoolAssetBalance);
   // Version 26 - 1.34
   app.component('RefreshButton', RefreshButton);
+  app.component('AssetBalanceStatisticSourceSetting', AssetBalanceStatisticSourceSetting);
```

Comment:
Do you think we could use a slot to pass this to the component instead?

Reply:
Probably we can, but we use <PremiumStatistics /> directly, so this slot will be really specific

Reply:
Hmm true, it would work fine for the details page but it would have to be passed through in the PremiumStatistics. 

Let's keep the PR's approach then thanks.